webapp-user/README.md

```diff
@@ -0,0 +1,56 @@
+# System-User Web Apps
+
+## Overview
+The System-User Service webapp is built on React.js and using a create-react-app architecture.
+The app uses the current LTS version of Node.js.
+
+## Pre-requisites
+- Install NVM
+Install the [Node Version Manager](https://github.com/creationix/nvm) that will allow a specific
+version of Node.js to be installed. Follow the installation instructions.
+
+- Install the latest stable Node.js and npm
+The latest stable (LTS) version of Node can be found on the [Node website](nodejs.org).
+```bash
+# Overview of available commands
+nvm help
+
+# Install the latest stable version
+nvm install lts/*
+
+# Select the version to use
+nvm ls
+nvm use lts/*
+```
+
+- Install the nodejs dependencies
+```bash
+cd webapp-user
+npm install
+```
+
+## Testing
+To run the tests for an app:
+```bash
+cd webapp-user
+npm test
```

Comment:
Yes, the test in package.json need to run 'npm run build-css' and then run the tests.